### ISSUE TITLE
The training settings are different from the released config files.

### ISSUE THREAD

**Issue by abc-def-g** [dvlab-research/Stratified-Transformer#48]
In Page 6 of the paper, it says that  "For ScanNetv2, we train for 600 epochs with weight decay and batch size set to 0.1 and 8 respectively". But I find that in the released config files, the network is trained by 100 epochs with a based learning rate of 0.006. Should I follow the config file to reproduce the results?

**Comment by X-Lai** [comment]
Please note that the actual training epochs should be `loop * epochs`. The released configs are exactly those we use to get the results in the paper.
